### PR TITLE
Hotfix/impact numbers

### PR DESCRIPTION
**Pull request by VirginiaDooley**
This PR updates the impact metrics following the 2024 General Election, includes a security patch for Django, and adjusts indentation.

- [x] Updates postcode search number
- [x] Removes number of reminders sent in 2019
- [ ] Updates more likely to vote

Screenshot 2024-10-10 at 9 14 25 AM
Screenshot 2024-10-10 at 9 14 09 AM

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207776388404082

### CODE REVIEW

File: democracy_club/templates/home.html [hunk 1]
```diff
@@ -26,8 +26,7 @@ <h2>Our impact in numbers</h2>
                 and local elections.</p>
 
             <ul class="ds-switcher ds-showcase" style="--gridCellMin: 16ch; --gridGap: 1.5rem;">
-                <li><strong>219m</strong> reminders to vote sent via partnerships in&nbsp;2019</li>
-                <li><strong>3.5m</strong> people used our services in the last general election</li>
+                <li><strong>14m</strong> people used our services in the 2024 general election</li>
                 <li><strong>83%</strong> of users say that our services make them more likely to&nbsp;vote</li>
```
Comment (VirginiaDooley) [inline democracy_club/templates/home.html:30]: @pmk01 Regarding "more likely to vote", with the addition of the "no change" option, this number has become more complicated to report. Do we want to revert to 'found useful'? If I average the numbers from WDIV and WCIVF, we are at 86%.

Comment (pmk01) [reply democracy_club/templates/home.html:30]: Yes, I agree, move back to 'found useful'.

Comment (pmk01) [reply democracy_club/templates/home.html:30]: The specific question is 'did you find what you were looking for?' Is there's a better way to summarise this?
"85% of users found the information they needed", or similar?

Comment (VirginiaDooley) [reply democracy_club/templates/home.html]: I've made this change.

**Review by pmk01 — APPROVED** [review]

**Pull request merged**